extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/dataaddress/http/HttpHeaderMapper.java

```diff
@@ -0,0 +1,46 @@
+package de.sovity.edc.ext.wrapper.api.common.mappers.dataaddress.http;
```

Comment:
Missing copyright header
```suggestion
/*
 *  Copyright (c) 2022 sovity GmbH
 *
 *  This program and the accompanying materials are made available under the
 *  terms of the Apache License, Version 2.0 which is available at
 *  https://www.apache.org/licenses/LICENSE-2.0
 *
 *  SPDX-License-Identifier: Apache-2.0
 *
 *  Contributors:
 *       sovity GmbH - initial API and implementation
 *
 */
 
package de.sovity.edc.ext.wrapper.api.common.mappers.dataaddress.http;
```